exporter.go

```diff
@@ -778,6 +785,16 @@ func (e *Exporter) extractInfoMetrics(ch chan<- prometheus.Metric, info string,
 			slaveInfoFields    = map[string]bool{"master_host": true, "master_port": true, "slave_read_only": true}
 		)
 
+		if strings.HasPrefix(fieldKey, "master_host") {
```

Comment:
From the snippet you posted in #371 it looks like this will occur in the `# Replication` section so maybe only pick these up while in the `fieldClass == "Replication"` section down starting line 810?

Reply:
Also, why `HasPrefix` and not match it exactly?

Reply:
I can simplify it.